ssg/templates.py

```diff
@@ -19,6 +19,88 @@
 # Callback functions for processing template parameters and/or validating them
 
 
+def accounts_password(data, lang):
+    if lang == "oval":
+        data["sign"] = "-?" if data["variable"].endswith("credit") else ""
+    return data
+
+
+def audit_rules_dac_modification(data, lang):
+    return data
+
+
+def audit_rules_file_deletion_events(data, lang):
+    return data
+
+
+def audit_rules_login_events(data, lang):
+    path = data["path"]
+    name = re.sub(r'[-\./]', '_', os.path.basename(os.path.normpath(path)))
+    data["name"] = name
+    if lang == "oval":
+        data["path"] = path.replace("/", "\\/")
+    return data
+
+
+def audit_rules_path_syscall(data, lang):
+    if lang == "oval":
+        pathid = re.sub(r'[-\./]', '_', data["path"])
+        # remove root slash made into '_'
+        pathid = pathid[1:]
+        data["pathid"] = pathid
+    return data
+
+
+def audit_rules_privileged_commands(data, lang):
+    path = data["path"]
+    name = os.path.basename(path).replace(r'[-\./]', '_')
+    data["name"] = name
+    if lang == "oval":
+        data["id"] = data["_rule_id"]
+        data["title"] = "Record Any Attempts to Run " + name
+        data["path"] = path.replace("/", "\\/")
+    return data
+
+
+def audit_rules_unsuccessful_file_modification(data, lang):
+    return data
+
+
+def audit_rules_unsuccessful_file_modification_o_creat(data, lang):
+    return data
+
+
+def audit_rules_unsuccessful_file_modification_o_trunc_write(data, lang):
+    return data
+
+
+def audit_rules_unsuccessful_file_modification_rule_order(data, lang):
```

Comment:
What about defining a function `def noop(data, lang): return lang` and using that `noop` wherever applicable?

Reply:
That would be awesome if it wouldn't be already done the other way.